master/buildbot/steps/shell.py

```diff
@@ -416,6 +416,8 @@ def __init__(self,
             self.directoryLeavePattern = directoryLeavePattern
         if suppressionFile:
             self.suppressionFile = suppressionFile
+        if suppressions:
+            self.suppressions = suppressions
```

Comment:
there is no default value for self.suppressions
so the start command should raise an exception in case of default suppressions = None